src/Helper.php

```diff
@@ -108,7 +108,9 @@ function (string $className) {
         $constructs = Classy\Constructs::fromDirectory($directory);
 
         $classyNames = \array_diff(
-            $constructs,
```

Comment:
This works fine because of https://github.com/localheinz/classy/blob/0.3.0/src/Construct.php#L33-L41, but tripped me when taking a look, so better to be explicit.